### PR TITLE
Encapsulate feeds.Package{} struct creation via constructor func

### PR DESCRIPTION
**Pull request by tom--pollard**
This also adds logging output per Package, inline with goproxy feed

### CODE REVIEW

File: feeds/feed.go [hunk 1]
```diff
@@ -12,3 +15,17 @@ type Package struct {
 	CreatedDate time.Time `json:"created_date"`
 	Type        string    `json:"type"`
 }
+
+func NewPackage(created, cutoff time.Time, name, version, feed string) *Package {
+	if created.Before(cutoff) {
+		return nil
+	}
+	log.Println("Processing:", name, version, feed)
```
Comment (Qinusty) [inline feeds/feed.go:23]: Consider using `log.WithFields(...)` here as seen in `cmd/scheduled-feed/main.go`.
```
log.WithFields(log.Fields{
	"name":         pkg.Name,
	"feed":         pkg.Type,
	"created_date": pkg.CreatedDate,
}).Print("sending package upstream")
```

Comment (tom--pollard) [reply feeds/feed.go:23]: Sure, also with that in mind it might also be a consideration to globally switch to `logrus` throughout

**Comment by tom--pollard** [comment]
Updated to include nuget & packagist

**Comment by tom--pollard** [comment]
Is this ok to merge, @jordan-wright @dlorenc ? :smiley:

**Review by Qinusty — APPROVED** [review]
Looks good @tom--pollard. ~~Only Nit is that I'm starting to think maybe it's cleaner if we handle cutoff in Scheduler.Poll() rather than inside the package creation. This moves responsibility away from the individual feeds and onto the polling service. Happy to open this as a separate issue/PR though.~~

Edit: Although... Some feed APIs may benefit from the cutoff as a query parameter... Let's leave it as is :)

**Pull request merged**